src/mui/form/SimpleForm.js

```diff
@@ -40,8 +44,11 @@ const ReduxForm = reduxForm({
     validate: validateForm,
 })(SimpleForm);
 
+const getInitialValues = once((children, record, defaultValue) =>
```

Comment:
Shouldn't this be extracted to `../form/getInitialValues` ? I see you use it in TabbedForm too